exercise/templates/exercise/_children.html

```diff
@@ -11,7 +11,17 @@
 {% endif %}
 {% elif entry|is_listed %}
 <li>
-  {% if accessible %}
+  {% if accessible and not entry.submittable%}
```

Comment:
Missing space after `entry.submittable`.

Reply:
Can you explain what these if statements do? I'd like to know what is used for chapters and exercises, and what happens before and after the opening time and reading opening time.

Reply:
`entry.submittable` checks if the children is exercise or chapter.